src/components/RootApp/ScalprumRoot.tsx

```diff
@@ -259,7 +258,6 @@ const ScalprumRoot = memo(
                   }
                 />
               )}
-              {ITLess() && <Route path="/insights/satellite" element={<SatelliteToken />} />}
```

Comment:
Removed the react router route in ScalprumRoot in favor of the exposed module approach.